modules/dxp/apps/saml/saml-opensaml-integration-test/src/testIntegration/java/com/liferay/saml/opensaml/integration/internal/field/expression/handler/test/DefaultUserFieldExpressionHandlerTest.java

```diff
@@ -0,0 +1,150 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.saml.opensaml.integration.internal.field.expression.handler.test;
+
+import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
+import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.kernel.model.User;
+import com.liferay.portal.kernel.service.UserLocalService;
+import com.liferay.portal.kernel.test.rule.AggregateTestRule;
+import com.liferay.portal.kernel.test.util.ServiceContextTestUtil;
+import com.liferay.portal.kernel.test.util.UserTestUtil;
+import com.liferay.portal.kernel.util.HashMapBuilder;
+import com.liferay.portal.test.rule.Inject;
+import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
+import com.liferay.saml.opensaml.integration.field.expression.handler.registry.UserFieldExpressionHandlerRegistry;
+import com.liferay.saml.opensaml.integration.processor.UserProcessor;
+import com.liferay.saml.opensaml.integration.processor.factory.UserProcessorFactory;
+
+import java.util.Map;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Tamas Biro
+ */
+@RunWith(Arquillian.class)
+public class DefaultUserFieldExpressionHandlerTest {
```

Comment:
We are not testing DefaultUserFieldExpressionHandler. The logic being tested is in that class but that may change. What we are testing is UserProcessor . We need to rename / move the test class.